stdlib/public/runtime/SwiftRT-ELF.cpp

```diff
@@ -65,6 +83,8 @@ static void swift_image_constructor() {
       SWIFT_SECTION_RANGE(swift5_assocty),
       SWIFT_SECTION_RANGE(swift5_replace),
       SWIFT_SECTION_RANGE(swift5_replac2),
+      SWIFT_SECTION_RANGE(swift5_builtin),
+      SWIFT_SECTION_RANGE(swift5_capture),
   };
 
 #undef SWIFT_SECTION_RANGE
```

Comment:
I think that the changes to this file can be split out into a standalone change.

Reply:
ok, will do

Reply:
https://github.com/apple/swift/pull/26479